lib/TimeoutException.php

```diff
@@ -7,6 +7,6 @@ class TimeoutException extends \Exception {
      * @param string|null $message
      */
     public function __construct(string $message = null) {
-        parent::__construct($message ?: "Promise resolution timed out");
+        parent::__construct($message ?: "Operation timed out");
```

Comment:
Dunno, Amp is _the_ Promise library. no need to be more specific here. If you catch, you shall feel free to `use Amp\TimeoutException as PromiseTimeoutException;`